fecgov/openFEC
# Add tsvector column to schedule A and schedule B aggregate data

## Summary (required)

- Resolves #5739 
Create two materialized views for schedule A aggregate data. 
Add tsvector column to ofec_sched_b_aggregate_recipient_mv

### Required reviewers
1-2 developers

## How to test
1. Download feature branch
2. Test migration by running  `flyway migrate`
3. Test refreshing materialized views.  Running the follow commands and make sure these two materialized views are being refreshed.
    ofec_sched_a_aggregate_employer_mv
    ofec_sched_a_aggregate_occupation_mv
```
dropdb` cfdm_test
createdb cfdm_test
invoke create_sample_db
```
